app/controllers/navigation/Navigator.scala

```diff
@@ -285,26 +286,30 @@ object Navigator {
 
   val standardCacheDependent: PartialFunction[DeclarationPage, (ExportsDeclaration, Mode) => Call] = {
     case CarrierDetails => carrierDetailsPreviousPage
+    case Document       => previousDocumentsPreviousPageDefault
```

Comment:
@SteveSugden Here you have Document in Navigator.

It's cache dependent thing so in separate methods

Reply:
Missed that.  Sorry!